frontend/src/utilities/segmentIOUtils.tsx

```diff
@@ -15,7 +15,49 @@ export const fireTrackingEventRaw = (eventType: string, properties?: any): void
       }`,
     );
   } else if (window.analytics) {
-    window.analytics.track(eventType, { ...properties, clusterID });
+    window.analytics.track(
+      eventType,
+      {
+        ...properties,
+        clusterID,
+      },
+      {
+        app: {
+          version: INTERNAL_DASHBOARD_VERSION,
+        },
+      },
+    );
+  }
+};
+
+export type IdentifyProps = {
+  anonymousID: string;
+  canCreateProjects: boolean;
+  canDelete: boolean;
+  canUpdate: boolean;
+  isAdmin: boolean;
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any,@typescript-eslint/explicit-module-boundary-types
+export const fireIdentifyEvent = (properties?: IdentifyProps): void => {
```

Comment:
What explicit any are you preventing?

Reply:
Leftover from when I started the method. Good catch.